pkg/morph/deploy/funds.go

```diff
@@ -448,3 +445,44 @@ func distributeNEOToAlphabetContracts(ctx context.Context, prm distributeNEOToAl
 		txMonitor.trackPendingTransactionsAsync(ctx, vub, mainTxID, fallbackTxID)
 	}
 }
+
+func divideFundsEvenly(fullAmount *big.Int, n int, f func(ind int, amount *big.Int)) {
+	if fullAmount.IsUint64() {
+		divideFundsEvenlyU64(fullAmount.Uint64(), n, func(ind int, amount uint64) {
+			f(ind, new(big.Int).SetUint64(amount))
+		})
+		return
+	}
+
+	quot := new(big.Int).Div(fullAmount, big.NewInt(int64(n)))
+	rem := new(big.Int).Mod(fullAmount, big.NewInt(int64(n)))
+
+	for i := 0; i < n; i++ {
+		amount := new(big.Int).Set(quot)
+		if rem.Sign() > 0 {
+			amount.Add(amount, big.NewInt(1))
+			rem.Sub(rem, big.NewInt(1))
+		} else if amount.Sign() == 0 {
+			return
+		}
+
+		f(i, amount)
+	}
+}
+
+func divideFundsEvenlyU64(fullAmount uint64, n int, f func(ind int, amount uint64)) {
```

Comment:
One version is enough. We can't have more than 100M NEO.

Reply:
done